.gitignore

```diff
@@ -2,3 +2,12 @@ bazel-*
 .bazelrc.user
 **/.terraform/*
 node_modules/
+
+# Don't commit lockfile for now as it is unstable. Do allow for it to be
```

Comment:
This isn't in 2.x? 🤔

Reply:
2.x is still on Bazel 6. MDOULE.bazel.lock generation didn't come until Bazel 7.